File: admin-gui/src/main/java/org/aktin/dwh/admin/importer/ImportState.java
package org.aktin.dwh.admin.importer;

// possible states of PropertyKey.state
public enum ImportState {
    upload_successful,
    verifying,
    verification_queued,
    verification_successful,
    verification_cancelled,
    verification_failed,
    importing,
    importing_queued,
    import_successful,
    import_cancelled,
    import_failed,
}

File: admin-gui/src/main/webapp/app/importer/ImportState.ts
export enum ImportState {
    ready = 'Bereit',
    uploading = 'Wird hochgeladen',
    upload_queued = 'In Warteschlange zum Hochladen',
    upload_cancelled = 'Hochladen abgebrochen',
    upload_failed = 'Hochladen fehlgeschlagen',
    upload_successful = 'Hochladen abgeschlossen',
    verifying = 'Wird geprüft',
    verification_queue = "In Warteschlange zur Prüfung",
    verification_cancelled = 'Prüfung abgebrochen',
    verification_failed = 'Prüfung fehlgeschlagen',
    verification_successful = 'Prüfung abgeschlossen',
    importing = 'Wird importiert',
    import_queued = "In Warteschlange zum Import",
    import_cancelled = 'Import abgebrochen',
    import_failed = 'Import fehlgeschlagen',
    import_successful = 'Import abgeschlossen'
}